src/Lib/Scanner.php

```diff
@@ -48,7 +48,7 @@ public static function all(): array
      * Return all templates for a given plugin.
      *
      * @param string $plugin The plugin to find all templates for.
-     * @return mixed
+     * @return string[]
      */
     public static function plugin(string $plugin)
```

Comment:
The return value can be type hinted as `array` now.

Reply:
I wasnt sure for a minor release on a public method :)
But then again this is a plugin, not core.

Reply:
I already added lot of missing typehints in a previous PR :)